requirements/constraints.txt

```diff
@@ -1 +1,3 @@
 -c common_constraints.txt
+
+alembic<1.11
```

Comment:
Can you add a comment about why since not everyone will look at the commit, and also a link to the ticket to come back and fix this when the upstream bug is fixed.

Reply:
Done